### ISSUE TITLE
Test failing due to HDF5 error

### ISSUE THREAD

**Issue by tomw66** [ADAQ-AQI/clean-air-project#50]
The issue with the test failing in tox seems to be (of course) related to conda. I've been adjusting package channels and rebuilding environments, but not sure this is fixible without abandoning anaconda entirely:
https://stackoverflow.com/questions/74226436/hdf5-error-when-opening-nc-files-in-python-with-xarray

Short of that, I've basically Ship of Theseus-ed the test to:
- be in a different location
- have a different name
- have a different description
- use completely different source data
- use remade timeseries data
- load a different path 
- test a different property

So basically an entirely new test - but it still fails, whilst all others are fine! I'm at a loss :(

_Originally posted by @tomw66 in https://github.com/ADAQ-AQI/clean-air-project/issues/49#issuecomment-1421018319_
